README.md

```diff
@@ -1,53 +1,74 @@
 
-## Usage
+## Usage Examples
+Use the **Run MATLAB Command** action to run MATLAB scripts, functions, and statements tailored to your specific needs. You can use this action to flexibly customize your test run or add a MATLAB-related step to your workflow. 
 
-You can use this action `with`:
-| Argument  | Description |
-|-----------|-------------|
-| `command` | (Required) Script, function, or statement to execute. <br/> If the value of `command` is the name of a MATLAB script or function, do not specify the file extension. If you specify more than one MATLAB command, use a comma or semicolon to separate the commands. <br/> **Example**: `myscript` <br/> **Example**: `results = runtests, assertSuccess(results);`
+### Run MATLAB Script on Self-Hosted Runner
+Use a self-hosted runner to run the commands in a file named `myscript.m` in the root of your repository.
 
-### Using a GitHub-hosted runner?
-If you are using a GitHub-hosted runner, you can use the [Set up MATLAB action](https://github.com/matlab-actions/setup-matlab/) to install MATLAB on the runner.
+```yaml
+name: Run MATLAB Script on Self-Hosted Runner
+on: [push]
+jobs:
+  my-job:
+    name: Run MATLAB Script
+    runs-on: self-hosted
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Run script
+        uses: matlab-actions/run-command@v0
+        with:
+          command: 'myscript'
+```
 
-## Example
+### Run MATLAB Commands on GitHub-Hosted Runner
+Use the [Set Up MATLAB](https://github.com/matlab-actions/setup-matlab/) action when you want to run MATLAB code or Simulink models on a GitHub-hosted runner. The action installs your specified MATLAB release (R2020a or later) on a Linux virtual machine. If you do not specify a release, the action installs the latest release of MATLAB.
+
+For example, install the latest release of MATLAB on a GitHub-hosted runner, and then use the **Run MATLAB Command** action to execute your MATLAB commands.
 
 ```yaml
-name: Sample workflow
+name: Run MATLAB Commands on GitHub-Hosted Runner
 on: [push]
-
 jobs:
   my-job:
-    name: Say hello from MATLAB
+    name: Run MATLAB Commands
     runs-on: ubuntu-latest
     steps:
-      # Set up MATLAB using this action first if running on a GitHub-hosted runner!
-      - uses: matlab-actions/setup-matlab@v0
-      
-      - name: Run "disp('hello world')" directly using MATLAB
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Install MATLAB
+        uses: matlab-actions/setup-matlab@v0
+      - name: Run commands
         uses: matlab-actions/run-command@v0
         with:
-            command: disp('hello world')
+          command: 'results = runtests, assertSuccess(results);'
 ```
 
+## Run MATLAB Command
+When you define your workflow in the `.github/workflows` directory of your repositoy, you can specify the **Run MATLAB Command** action using the `run-command` key. The action requires an input.
```

Comment:
In practice, I don't think users will be able to call `run-command` without qualifying it with our org and a version 🤔 
They'd have to use `matlab-actions/run-command@v0` 🤔

Reply:
How about the updated version?